### PR TITLE
feat: Displayed failed tests + breadcrumb improvment

### PR DESCRIPTION
**Pull request by sguiheux**

### CODE REVIEW

File: ui/src/assets/i18n/en.json [hunk 1]
```diff
@@ -70,13 +70,17 @@
   "common_applications" : "Applications",
   "common_artifact" : "Artifact",
   "common_artifacts" : "Artifacts",
+  "common_author" : "Author: ",
```
Comment (fsamin) [inline ui/src/assets/i18n/en.json]: _title

File: ui/src/assets/i18n/en.json [hunk 2]
```diff
@@ -70,13 +70,17 @@
...
+  "common_branch" : "Branch: ",
```
Comment (fsamin) [inline ui/src/assets/i18n/en.json]: _title

File: ui/src/assets/i18n/en.json [hunk 3]
```diff
@@ -70,13 +70,17 @@
...
   "common_close" : "Close",
   "common_commit" : "Commit",
   "common_commits" : "Commits",
+  "common_commit_title" : "Commit: ",
   "common_date_end" : "End",
   "common_date_start" : "Start",
   "common_description" : "Description",
   "common_download" : "Download",
+  "common_duration" : "Duration: ",
```
Comment (fsamin) [inline ui/src/assets/i18n/en.json]: _title

File: ui/src/assets/i18n/fr.json [hunk 4]
```diff
@@ -70,13 +70,17 @@
   "common_applications" : "Applications",
   "common_artifact" : "Artefact",
   "common_artifacts" : "Artefacts",
+  "common_author" : "Auteur : ",
```
Comment (fsamin) [inline ui/src/assets/i18n/fr.json]: _title

File: ui/src/assets/i18n/fr.json [hunk 5]
```diff
@@ -70,13 +70,17 @@
...
+  "common_branch" : "Branche : ",
```
Comment (fsamin) [inline ui/src/assets/i18n/fr.json]: _title

File: ui/src/assets/i18n/fr.json [hunk 6]
```diff
@@ -70,13 +70,17 @@
...
   "common_close" : "Fermer",
   "common_commit" : "Changement",
   "common_commits" : "Changements",
+  "common_commit_title" : "Changement : ",
   "common_date_end" : "Date de fin",
   "common_date_start" : "Date de début",
   "common_description" : "Description",
   "common_download" : "Télécharger",
+  "common_duration" : "Durée : ",
```
Comment (fsamin) [inline ui/src/assets/i18n/fr.json]: _title

File: ui/src/assets/i18n/en.json [hunk 7]
```diff
@@ -156,6 +161,14 @@
   "pipeline_type" : "Type of pipeline",
   "pipeline_updated" : "Pipeline updated",
 
+  "pipeline_run_test_total_title_s" : " tests",
+  "pipeline_run_test_total_title_s" : " test",
```
Comment (yesnault) [inline ui/src/assets/i18n/en.json]: ?

File: ui/src/assets/i18n/en.json [hunk 8]
```diff
@@ -156,6 +161,14 @@
...
+  "pipeline_run_test_failed_title_s" : " tests failed",
+  "pipeline_run_test_failed_title" : " test failed",
+  "pipeline_run_test_skipped_title_s" : " tests skipped",
+  "pipeline_run_test_skipped_title_s" : " test skipped",
```
Comment (yesnault) [inline ui/src/assets/i18n/en.json]: idem

**Pull request merged**